### ISSUE TITLE
Define roles and skillsets needed

### ISSUE THREAD

**Issue by sethetter** [openwichita/openbudgetwichita#1]
What kind of technical and non-technical skills will be necessary to complete this project?

From what I've heard from @mkchandler and the OKC crew, we need someone with D3.js experience.

Getting someone involved with budget / finance expertise could also prove useful. We'll be working on getting this involvement from the city, but someone from the community would be helpful as well.

What else?

**Comment by br0xen** [comment]
I've got some JS experience. I think I messed with D3.js at one point, but not enough to claim experience in it.

**Comment by curtism99** [comment]
I have JS experience, but have never seen or worked with D3.js before.

**Comment by Mearnest** [comment]
Same as br0xen.

**Comment by mjvandecreek** [comment]
I've used some libraries which wrap D3 but not much direct contact.
i.e.
http://nvd3.org/

Nice rap list here:
https://github.com/wbkd/awesome-d3

This past year, I evaluated about 12 dataviz libraries over last year at work.

I'd be good later on to critic the UX and purpose of the data views.
(some of the oakland ux bugged me)

And a hype video from a viz-lab:
http://idl.cs.washington.edu/papers/d3
